doc/tor.md

```diff
@@ -95,12 +95,18 @@ Starting with Tor version 0.2.7.1 it is possible, through Tor's control socket
 API, to create and destroy 'ephemeral' hidden services programmatically.
 Bitcoin Core has been updated to make use of this.
 
-This means that if Tor is running (and proper authorization is available),
-Bitcoin Core automatically creates a hidden service to listen on, without
-manual configuration. This will positively affect the number of available
-.onion nodes.
+This means that if Tor is running (and proper authentication has been configured),
+Bitcoin Core automatically creates a hidden service to listen on. This will positively 
+affect the number of available .onion nodes.
 
 This new feature is enabled by default if Bitcoin Core is listening, and
 a connection to Tor can be made. It can be configured with the `-listenonion`,
 `-torcontrol` and `-torpassword` settings. To show verbose debugging
 information, pass `-debug=tor`.
+
+Connecting to Tor's control socket API requires one of two authentication methods to be 
+configured.  The most straightforward is the use of the `-torpassword` flag and a `hash-password` 
+which can be enabled in Tor configuration.  Alternatively the user running bitcoind must have 
+write access to the `CookieAuthFile` specified in Tor configuration.  This can be achieved 
+by adding both the user running tor, and the user running bitcoind to the same group and setting 
+permissions appropriately.
```

Comment:
Note that most distributions already have something along those lines, e.g. on Debian-based there is a debian-tor group, and you make cookie auth work by adding users who need it to the debian-tor group. I'd reword this to just say that you need to make sure the bitcoind user is part of the Tor group.

Anyone know how non-Debian distros do this?